packages/@aws-cdk/aws-eks/README.md

```diff
@@ -232,8 +232,42 @@ cluster.addNodegroupCapacity('extra-ng-spot', {
 
 #### Launch Template Support
 
-You can specify a launch template that the node group will use. Note that when using a custom AMI, Amazon EKS doesn't merge any user data.
-Rather, You are responsible for supplying the required bootstrap commands for nodes to join the cluster.
+You can specify a launch template that the node group will use. For example, this can be useful if you want to use
+a custom AMI or add custom user data.
+
+Since the custom user data is merged with the Amazon EKS user data, it must be encoded in the MIME multi-part
+archive format. Visit the [Launch Template Docs](https://docs.aws.amazon.com/eks/latest/userguide/launch-templates.html#launch-template-user-data)
+for insights.
```

Comment:
```suggestion
When supplying a custom user data script, it must be encoded in the MIME multi-part archive format, since Amazon EKS merges with its own user data. Visit the [Launch Template Docs](https://docs.aws.amazon.com/eks/latest/userguide/launch-templates.html#launch-template-user-data)
for mode details.
```